packages/flutter_tools/lib/src/asset.dart

```diff
@@ -147,16 +148,21 @@ class _ManifestAssetBundle implements AssetBundle {
     }
 
     final String assetBasePath = globals.fs.path.dirname(globals.fs.path.absolute(manifestPath));
-
-    final PackageMap packageMap = PackageMap(packagesPath, fileSystem: globals.fs);
+    final PackageConfig packageConfig = await loadPackageConfigUri(
+      globals.fs.file(packagesPath).absolute.uri,
+      preferNewest: false,
```

Comment:
Lots of our test setups don't create .package_config files yet, so this will automatically fail looking for a .dart_tool directory